tests/test_webhook.py

```diff
@@ -452,6 +453,11 @@ def message_content(event):
         def message_sticker(event):
             self.assertEqual('message', event.type)
             self.assertEqual('sticker', event.message.type)
+            self.assertIn(
```

Comment:
[FYI]
From [documentation](https://developers.line.biz/en/reference/messaging-api/#wh-sticker), sticker resource types may be added without announcing. 

[IMO]
@okue 
This implementation is good enough, but how about the following checking to prevent some confusion?

```py
self.assertEqual('STATIC', event.message. sticker_resource_type)
```

Reply:
I think `assertIn` is better than `assertEqual`.
Since this method is decorated by `handler.add(MessageEvent, message=StickerMessage)`, thus event is not always `STATIC`.

Reply:
This check is unnecessary since this unit test is for `WebhookHandler`.

Reply:
I change my mind. https://github.com/line/line-bot-sdk-python/pull/229#discussion_r355872811